Tools/gstreamer/jhbuild.modules

```diff
@@ -156,6 +106,14 @@
             hash="sha256:68ca8d071dcb84ac82e3c6d7f02a55937d9f690fcb215853f4aa1de8d459812f" />
   </meson>
 
+  <meson id="sparkle-cdm" mesonargs="-Dsample-player=disabled">
+    <branch repo="github-tarball"
+            module="Sparkle-CDM/sparkle-cdm/tarball/${version}"
+            checkoutdir="sparkle-cdm"
+            version="3e1e0dcb221d044213f8c6e32b0b06102e7fb224"
```

Comment:
I think the version is supposed to be numeric, so I would omit this. What it means is "if this version or a greater version is already installed as a system dependency, then do not download and build." This doesn't work for git hashes.

Reply:
You can just copy the hash into the `module=` line instead

Reply:
AFAIK that version check is only performed when the entry has a `<pkg-config>` entry.
I tried removing the version and moving it into the module but then JHBuild fails to parse the xml, seems it expects a version number.
So I will just provide a version number based on the commit date (20231106)